.storybook/preview.js

```diff
@@ -1,11 +1,160 @@
+// Decanter Styles.
+import 'decanter/dist/decanter.css'
+import 'decanter/dist/decanter.js'
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
+import { addDecorator } from '@storybook/react'
+import { withTests } from '@storybook/addon-jest'
+import results from './.jest-test-results.json';
 
+// Our Breakpoints
+const customViewports = {
```

Comment:
Added some of our breakpoints and a misc height for testing.

Reply:
This is nice